### ISSUE TITLE
Unable to run the repo 

### ISSUE THREAD

**Issue by inkerton** [GDSC-RCCIIT/gdg-website#6]
![image](https://github.com/user-attachments/assets/7d91ab8d-c30d-4f58-b068-ac776ccc7c19)

![image](https://github.com/user-attachments/assets/341095e6-306f-4f5c-a86d-71cf64ca7e5c)

What is the issue here? can you guide me how to run the rpo i would like to see and contribute @danger-ahead @rishicds

**Comment by rishicds** [comment]
Hello, this issue is because of npm modules getting corrupted, try deleting the node_modules folder, and run npm i, alternatively you can also run `npm cache clean --force` then run npm i, let me know if it fixed the issue @inkerton

**Comment by inkerton** [comment]
![image](https://github.com/user-attachments/assets/640a4fb6-9741-4447-9f3f-aa30448b497f)


@rishicds I am still getting the same error

**Comment by inkerton** [comment]
@rishicds  Okay Got it working Can you please specify if this repo is still under GSSOC ?

If Yes, then I would like to work on it I will shere a link to my work 
This is how the page was before
![image](https://github.com/user-attachments/assets/503e1788-1a48-4464-bdc7-6e0fe365f28e)



<hr/>
And this is my work 


https://github.com/user-attachments/assets/d5e08e33-661b-40cc-adab-4ba1d6119642

**Comment by inkerton** [comment]
@rishicds  If this repo is still under GSSOC? Tell me how would you like to proceed with the work???

**Comment by rishicds** [comment]
> @rishicds  If this repo is still under GSSOC? Tell me how would you like to proceed with the work???

Repo will be reinstated in GSSOC in a few days

**Comment by rishicds** [comment]
> @rishicds  If this repo is still under GSSOC? Tell me how would you like to proceed with the work???


You can create issues on what parts of the website you would like to build.

**Comment by inkerton** [comment]
So can I start working now? 
All the PRs I do will be given points after its reinstated or right now. 

I would like to start with building a basic landing page first.

**Comment by rishicds** [comment]
> @rishicds 
> So can I start working now? 
> All the PRs I do will be given points after its reinstated or right now. 
> 
> I would like to start with building a basic landing page first.

Yes you can start

**Comment by inkerton** [comment]
@rishicds I asked you 2 days ago if i can start working on the landing page yet You are now holding off my PRs assigning it to somebody who wasn't active for the past week

**Comment by rishicds** [comment]
@inkerton issues are assigned on first come first serve basis, that person has replied with a design, you should not start working on a pr unless it has been assigned to you, if they dont give me an update in 3 days I will assign that issue to you. :)

**Comment by inkerton** [comment]
I asked you clearly though if I can start working if you don't read the comment properly or were inactive due to some reason then I don't think it is my fault. And if assigning the issue was such a big thing then Why didn't you assign the task to the person a week ago when the first commented or you shouldn't have said yes. 

As an admin I don't think I made a mistake by asking you if I can start working. You should have checked before telling me to start .

That issue was clearly not assigned to anyone when I asked you or when I made another issue or when I made the PR. 
But from now on I won't trust the words or comments this was a really frustrating part for me. Assign me the issue from now on and be responsive as much as you can. That would be great.

![image](https://github.com/user-attachments/assets/d0368c3f-2c55-46af-8bfa-cedfed7d56df)

**Comment by rishicds** [comment]
I apologise for this oversight, if the other person who was assigned doesn't give a pr in 2 days I'll merge your prs, don't worry.
